public/app/plugins/datasource/tempo/graphTransform.ts

```diff
@@ -146,7 +146,7 @@ export const errorRateMetric = {
 };
 export const durationMetric = {
   expr: 'histogram_quantile(.9, sum(rate(traces_spanmetrics_duration_seconds_bucket{}[$__range])) by (le))',
-  params: ['span_status="STATUS_CODE_ERROR"'],
+  params: [],
```

Comment:
i assume the filter params are added elsewhere?

Reply:
Yeah, in the buildExpr function. Helps me to merge params, filters, and __data.fields for the query :)